spec/controllers/catalog_controller_spec.rb

```diff
@@ -151,13 +162,32 @@
 
       let(:upd_version) { 4 }
 
-      it 'updates the version' do
-        expect(comp_moab.version).to eq upd_version
+      it 'updates CompleteMoab#version' do
+        expect(comp_moab.reload.version).to eq upd_version
+      end
+
+      it 'updates PreservedObject#current_version' do
+        expect(pres_obj.reload.current_version).to eq upd_version
       end
 
       it 'returns an ok response code' do
         expect(response).to have_http_status(:ok)
       end
+
+      context 'updating a non-primary' do
+        let!(:pres_obj) { create(:preserved_object, druid: bare_druid, current_version: 1) } # as if the one on srA was always primary
+        let!(:comp_moab) { create(:complete_moab, preserved_object: pres_obj, version: 3) } # create fixture_sr1 record, not created w/ PO this case
+        let(:primary_moab) { comp_moab_sr_a } # but we're still doing PATCH on the fixture_sr1 moab
+
+        it 'updates CompleteMoab#version' do
+          pending('this is known to fail, because CMH does not update CompleteMoab if its version does not match parent PO#current_version')
+          expect(comp_moab.reload.version).to eq upd_version
```

Comment:
will ticket this as a bug (the failure is that the `CompleteMoab` record for this druid on this storage root doesn't get its `#version` updated to 3 as it should).  not ideal, but also we don't really have an expected use case where a non-primary is of a higher version than the primary, as is the case here.  and in the opposite case (as seen in other tests), the primary gets its `#version` updated properly, because it should stay in sync with the parent `PreservedObject` (i _think_ maybe the lack of sync between the non primary CM and its parent PO is causing the trouble here).